Dockerfile

```
@@ -30,7 +14,7 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
 RUN echo 'deb https://download.docker.com/linux/debian bookworm stable' >> /etc/apt/sources.list
 RUN apt-get update && apt-get install docker-ce -y --no-install-recommends
 RUN git config --global user.name "ocaml" && git config --global user.email "ci"
-WORKDIR /var/lib/ocurrent
+WORKDIR /
```

```suggestion
WORKDIR /var/lib/ocurrent
```
We should not change this line.

This `WORKDIR` becomes the current working directory in the Docker container at runtime.  This is the mount point we use for the data volume, storing the database and job logs, etc.

---

Got it. I can't recall why I changed this before, and I've reverted it.